plugins/infinispan93/src/main/java/org/radargun/service/Infinispan93HotrodService.java

```diff
@@ -43,6 +57,20 @@ protected ConfigurationBuilder getDefaultHotRodConfig() {
       if (transactionMode != null) {
          config.transaction().transactionMode(TransactionMode.valueOf(transactionMode));
       }
+
+      config.maxRetries(maxRetries).socketTimeout(connectionTimeout).connectionTimeout(connectionTimeout);
+
+      if (propertiesPath != null) {
+         Properties p = new Properties();
+         try (Reader r = new FileReader(propertiesPath)) {
+            p.load(r);
+            config.withProperties(p);
+         } catch (FileNotFoundException e) {
+            e.printStackTrace();
+         } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Should we throw IllegalStateException?

Reply:
nit: if the `catch` are doing the same stuff, probability would be better combine them with `FileNotFoundException | IOException`

Reply:
@gustavolira @diegolovison it was not possible to combine them, as FileNotFoundException extends IOException (java didn't allow it), so I am just catching IOException and throwing IllegalStateException with message. Hope this ok for both of you :)

Reply:
Sorry Anna I didn't notice that